.changeset/forty-dots-jog.md

```diff
@@ -0,0 +1,7 @@
+---
+"@logto/next": major
+---
+
+remove explicit crypto module imports since Node now has global WebCrypto variable by default
```

Comment:
@wangsijie can you validate this? (check if samples work as expected)

Reply:
It works fine.